R/BASiCS_DiagPlot.R

```diff
@@ -5,54 +5,54 @@
 #' for more details on this diagnostic measure.
 #'
 #' @param object an object of class \code{\linkS4class{BASiCS_Summary}}
-#' @param Parameter Optional name of a chain parameter to restrict the histogram;
-#' if not supplied, all parameters will be assessed.
-#' Possible values: \code{'mu'}, \code{'delta'}, \code{'phi'},
-#' \code{'s'}, \code{'nu'}, \code{'theta'}, \code{'beta'},
-#' \code{'sigma2'} and \code{'epsilon'}. Default \code{Parameter = 'mu'}
+#' @param Parameter Name of the parameter to be plotted.
+#' Default \code{Parameter = 'mu'}
 #' @param Measure Character scalar specifying the diagnostic measure to plot.
-#' Current options are effective sample size and the Geweke diagnostic
-#' criterion.
-#' @param x,y Optional MCMC parameter values to be plotted on the x or y axis, 
+#' Current options are effective sample size, the Geweke diagnostic
+#' criterion, and the $\hat{R}$ diagnostic, see \code{\link[posterior]{rhat}}.
```

Comment:
```suggestion
#' criterion, and the \code{R.hat} diagnostic, see \code{\link[posterior]{rhat}}.
```

Reply:
It's posterior::rhat rather than R.hat, but good shout